pubsub/kafka/kafka.go

```diff
@@ -122,6 +122,10 @@ func NewKafka(logger logger.Logger) pubsub.PubSub {
 	}
 }
 
+func (p *PubSub) GetKafka() *kafka.Kafka {
```

Comment:
Only methods related to the PubSub interface should act on this struct here.

Reply:
I wanted a way of exposing the internal common implementation of the kafka component for users of this repo as a library, given I added `func (k *Kafka) GetClient() sarama.Client` I was looking for a way of accessing it

Reply:
is there any other approach for achieving this that would be accepted? i.e the entire pubsub kafka implementation could be moved to `common/component/kafka/kafka.go` equivalent to [postgres](https://github.com/dapr/components-contrib/blob/main/state/postgresql/v1/postgresql.go#L24)

which then allows for 
```
st := NewPostgreSQLStateStore(...)
pgst := st.(*postgresql.PostgreSQLQuery)
pgst.GetDB()
...
```

Reply:
IMHO I think adding a "private" method is fine. We do that in other components and it can be useful for certification tests